notebooks/learning_graph_structured_data_h2.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>Great! Maybe add some more legend here? What does each point represent, what do the colors represent? It probably comes after, but the gif is hard to read by itself for a newcomer.</p><p> </p><p>Also:</p><ul><li>for some reason the countours of the graph have a yellow-ish color: is this expected?</li><li>the disk appears as an ellipse, it would be cleaner if it could be a real disk.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
@Hzaatiti What script did you use to generate those? I can send you some code for this as well.

Reply:
@Hzaatiti All that remains now for the plotting is to fix the yellowing for the figures and gif.